app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -64,6 +67,12 @@ class BrowserFragment : WebFragment(), CursorEvent {
         initSession()
     }
 
+    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?): View? {
```

Comment:
nit: I think it's more appropriate to override `onViewCreated`, which is called after `onCreateView` when the view hierarchy is guaranteed to be created.

tbh, I don't know it's separated (fuck fragments) – maybe so sub-class can inflate a layout, declaring final, but you can still get a callback? Or maybe it has to do with killing the Fragment's view hierarchy and reviving it without killing the whole Fragment?